styleguide/source/_patterns/00-atoms/10-forms/03-select-menu.twig

```diff
@@ -1,6 +0,0 @@
-<select name="{{ for|default('select') }}">
-  <option value="" selected="selected">{{ selectedContent|default('Select an option') }}</option>
```

Comment:
We need to retain this functionality. The menu should start with "Select an Option" instead of the first item.